package.json

```diff
@@ -72,6 +72,7 @@
         "test": "jest --silent",
         "test:watch": "jest --watchAll",
         "test:server": "node ./test/server.js",
+        "prepublish": "npm run build",
         "deploy": "npm test && npm run build && npm publish"
```

Comment:
This code is duplicated. `npm run build`. So deploy execute build twice. I think we don't need this command.

Reply:
You are right that `npm run build` is unnecessary here since it will be executed by the `prepublish` script.

Reply:
Amended and force pushed.

Reply:
What is the purpose of the pre-publish?

Reply:
It runs before `publish` script [[docs](https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-operation-order)]. It makes sure that `npm publish` will always transpile the latest `lib` folder.

Reply:
@gurisko It is deprecated.
![image](https://user-images.githubusercontent.com/10627668/100232866-49704500-2f6c-11eb-9e00-4322beb303af.png)

Reply:
@hg-pyun Very good point! I replaced it with `prepare` as also suggested in the screenshot you attached.